frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -711,6 +777,9 @@ <h1 class="page-header">Viewing Hijack <small id="hijack_status"></small></h1>
                 }else if(data['under_mitigation'] == true || data['withdrawn'] == true || data['outdated'] == true){
                     $("#mitigation_button").prop("disabled", true);
                 }
+                if(data['active'] == false){
+                    $("#learn_rule_button").prop("disabled", true);
```

Comment:
this does not exist anymore right?

Reply:
yes

Reply:
ok then it needs to be deleted. See next comment for checking the active flag to decide how to proceed next.